src/cdk-experimental/combobox/combobox.ts

```diff
@@ -103,6 +109,48 @@ export class CdkCombobox<T = unknown> implements OnDestroy, AfterContentInit {
     this.panelValueChanged.complete();
   }
 
+  onClick() {
```

Comment:
It would probably make sense to make one method like `_handleInteraction(action)` and then set up the event handlers like `'(click)': '_handleInteraction("click")'`, etc.